third_party/parser/cc/grammars/typedruby.ypp

```diff
@@ -2769,6 +2772,16 @@ opt_block_args_tail:
                       auto lvar = driver.build.accessible(self, pid);
                       $$ = driver.build.pin(self, $1, lvar);
                     }
+                | tCARET nonlocal_var
+                    {
+                      auto non_lvar = driver.build.accessible(self, $2);
+                      $$ = driver.build.pin(self, $1, non_lvar);
+                    }
+       p_expr_ref: tCARET tLPAREN expr_value tRPAREN
```

Comment:
Looking at the other entries we spaced them so that the `tCARET` part was alligned with the previous node which is `p_var_ref: tCARET tIDENTIFIER`. Is that not the right approach?

Reply:
I think the style is that the `:` for rules and the `|` for alternatives all should be in the same column (which might be the same thing as what you were saying).

Sorry for munging your code!  It looked like the indentation was wrong relative to something else, but I took the wrong path here.  I'll fix things up.

Reply:
No worries Nathan. I also wasn't 100% sure 🙂